test/source/api/core_test.cpp

```diff
@@ -374,39 +362,28 @@ TEST(Core, StoreTransactionsEmpty) {
 }
 
 TEST(Core, StoreTransactionsInvalidTrytes) {
-  IOTA::API::Core api(get_proxy_host(), get_proxy_port());
```

Comment:
any reason to remove this test?

Reply:
Well, StoreTransactionsInvalidTrytes and StoreTransactionsAlmostInvalidTrytes are the same test.
And StoreTransactionsAlmostInvalidTrytes was actually not tested the way it was intended to.
Both of them were checking api.storeTransactions({ "INVALIDTRYTES" })